### ISSUE TITLE
EAL: unsupported cpu type

### ISSUE THREAD

**Issue by glimchb** [opiproject/opi-poc#262]
```
spdk_1             | ERROR: This system does not support "RDSEED".
spdk_1             | Please check that RTE_MACHINE is set correctly.
spdk_1             | EAL: FATAL: unsupported cpu type.
spdk_1             | EAL: unsupported cpu type.
```
https://github.com/opiproject/opi-poc/runs/7662075562?check_suite_focus=true

**Comment by glimchb** [comment]
another https://github.com/opiproject/opi-poc/runs/7673092621?check_suite_focus=true

**Comment by glimchb** [comment]
also here https://github.com/opiproject/opi-poc/runs/7675054428?check_suite_focus=true

**Comment by seroyer** [comment]
Digging into this a bit more I thought that `skylake` was old enough that anything we ran on would have at least that capability.  Skylake has RDSEED.  It appears that the runners where this is failing do not have RDSEED.  We can try an older arch.  At a minimum, I know that dpdk (since 17.08 and pulled in by spdk) requires sse4.2 since which was introduced in `nehalem` which does not have RDSEED.  We can try that and see if `nehalem` is sufficiently new for current dpdk.

**Comment by seroyer** [comment]
I'm going to close this as fixed by #271 .  We can reopen if we see this again.
